ivy/functional/ivy/device.py

```diff
@@ -1079,7 +1083,7 @@ def at_dev(self, device):
         """
         return ivy.nested_map(
             self._data,
-            lambda x: x[device] if isinstance(x, MultiDevItem) else x,
```

Comment:
the `lambda` bindings here are referring to backend functions like such (https://github.com/unifyai/ivy/blob/d80b093cccdbf5baf35bc6749f939998c130a445/ivy/functional/backends/mxnet/elementwise.py#L242)

for the open contributing task, please choose a function to reformat from the lists here: https://github.com/unifyai/ivy/issues?q=is%3Aopen+is%3Aissue+label%3AToDo+label%3A%22Function+Reformatting%22 

thanks!